qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data_get
+  file = File.new('data.txt')
+  file.read
+end
+
+def weather_data(town, data)
+  data.split("\n").each do |i|
+    if i.include? town
+      str = i.scan(/\d*\.\d/).map(&:to_f)
+      return str
+    end
+  end
+end
+
+def mean(town, data)
+  return -1 unless data.match "#{town}:"
+
+  weather_data(town, data).sum / 12
+end
+
+def variance(town, data)
+  return -1 unless data.match "#{town}:"
```

Comment:
since you're getting your data in the `weather_data`, this operation is redundant